core/object/object.cpp

```diff
@@ -882,6 +917,18 @@ void Object::set_script(const Variant &p_script) {
 	Ref<Script> s = p_script;
 	ERR_FAIL_COND_MSG(s.is_null() && !p_script.is_null(), "Invalid parameter, it should be a reference to a valid script (or null).");
 
+	// Check if there is a custom type script present. If there is, then "script" should never be null
```

Comment:
```suggestion
	// Check if there is a custom type script present. If there is, then "script" should never be null.
```
Per the [comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide).